packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -451,7 +454,20 @@ impl PostgresDatabase {
                 .change_context(InsertionError)?;
         }
 
-        // TODO: Store link references
+        // FIXME: Without `collect` we get a weird lifetime error
+        let entity_type_references = entity_type
+            .inner()
+            .link_references()
+            .into_iter()
+            .map(|(_link_type_uri, entity_type_uri)| entity_type_uri)
+            .collect::<Vec<_>>();
+
+        // TODO: Store references in the database
```

Comment:
Why are we not doing this? I can do it in my PR if you'd like

Reply:
This requires a table, which maps an entity type to a link type. Currently, we don't have CRU operations on link types. I think this will be blocked by your PR?

Reply:
There are two sets of references we need to persist. The ones you have here will go in a table called `entity_type_entity_type_links`, which does not need link types. But I have code for it in my PR anyway, no worries.